### ISSUE TITLE
Create Registration Form

### ISSUE THREAD

**Issue by vishesh147** [MSTC-DA-IICT/Hacktoberfest22-E-Learn-App-Django#5]
Create a registration form HTML template using bootstrap. Must include option to register as a teacher or student and all necessary attributes according to their models.

**Comment by PicantaData** [comment]
Yes, a registration form that allows a user to register as a student or as a teacher. Could you briefly explain your idea of approach?

**Comment by PicantaData** [comment]
Yes, a registration form that allows a user to register as a student or as a teacher. 
Work on it!

**Comment by shruti2403** [comment]
As it asks user to register as either student or teacher, So firstly
proceeding with the selection page whether they want to register as student
or teacher. Then depending on whether they selected student or teacher,
individual forms will be opened.
I have decided to approach in this way.

Any suggestions if I missed something?

**Comment by PicantaData** [comment]
Yes, you may start working on that!

**Comment by ykodwani01** [comment]
Can I work on this issue if @shruti2403 is not going to continue?
![image](https://user-images.githubusercontent.com/94884744/197379695-1401925a-01b3-47e4-b051-91fe8e2cf4a3.png)
Here is the registration for that I have already made.

**Comment by shruti2403** [comment]
I am working on it, and it's almost ready..
